test/spies.js

```diff
@@ -354,4 +364,30 @@ describe('Chai Spies', function () {
       }).should.throw(chai.AssertionError, /to have been always called with exactly/);
     });
   });
+
+  describe('spy object', function () {
+    it('should create spy object with specified method names', function () {
+      var object = chai.spy.object('array', [ 'push', 'pop' ]);
+
+      object.push.should.be.a.spy;
+      object.pop.should.be.a.spy;
+    });
+
+    it('should create spy object with specified method definitions', function () {
+      var object = chai.spy.object('array', {
+        push: function() {
+          return 'push';
+        }
+      });
+
+      object.push.should.be.a.spy;
+      object.push().should.equal('push');
+    });
+
+    it('should create spy object without name', function () {
+      var object = chai.spy.object([ 'push' ]);
+
+      object.push.should.be.a.spy;
+    });
+  });
```

Comment:
Ok, one final thing - it'd be nice to see a test that utilises the `chai.spy.object(obj)`. Then I'm definitely going to merge :smile: